ipatests/prci_definitions/nightly_rawhide.yaml

```diff
@@ -6,23 +6,23 @@ topologies:
   master_1repl: &master_1repl
     name: master_1repl
     cpu: 4
-    memory: 5750
+    memory: 6450
   master_1repl_1client: &master_1repl_1client
     name: master_1repl_1client
     cpu: 4
-    memory: 6700
+    memory: 7400
   ipaserver: &ipaserver
     name: ipaserver
     cpu: 2
     memory: 4800
```

Comment:
This one can be lowered to 2400 as in nightly_master.yaml

Reply:
right, thanks flo.